crates/nu-command/src/filters/default.rs

```diff
@@ -43,8 +43,18 @@ impl Command for Default {
         call: &Call,
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
+        let default_value: Value = call.req(engine_state, stack, 0)?;
+        let columns: Vec<String> = call.rest(engine_state, stack, 1)?;
         let empty = call.has_flag(engine_state, stack, "empty")?;
-        default(engine_state, stack, call, input, empty)
+        default(
+            engine_state,
+            stack,
+            call,
+            input,
+            default_value,
+            empty,
+            columns.as_slice(),
```

Comment:
```suggestion
            columns,
```

We can pass ownership of the `Vec` to `default` for usage in the `.map` closure later